### PR TITLE
To Do React tutorial PR 

### PR DESCRIPTION
**Pull request by codehass**
### In this pull request : 
- Set up Gitflow
- Create React app
- Add linters
- Add components.
- Implement add, edit and delete functionality.

Thank you for reviewing my PR  👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,258 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<!-- > Add a link to your deployed project. -->
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
```
Comment (Wahaj-Ali) [inline README.md]: Kindly add the link to the live demo, if not available then you can remove this section from your Readme file to make it look impressive.

File: README.md [hunk 2]
```diff
@@ -1 +1,258 @@
...
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (Wahaj-Ali) [inline README.md]: Please make sure to add License here.

**Review by Wahaj-Ali — CHANGES_REQUESTED** [review]
Dear @codehass Your project is not working. It is giving the following errors:
Screenshot 2023-01-17 202840

**Please make sure to eliminate these errors and request for a re-review.**

- Linter errors fixed ❌ 
- Correct GitFlow followed ✅ 
- Best practices used  for HTML, CSS, JavaScript ✅
-  Descriptive commit history ❌ 

**You can also look for the in-line comments below to make further changes.**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on Slack

**Comment by codehass** [comment]
Thank you @Wahaj-Ali for all your feedback 😁 I will add all the changes. 👌

**Review by Wahaj-Ali — APPROVED** [review]
Hi @codehass 
### **Status: Approved**
Your project is complete and working fine 💯 
It's time to merge 🎉

well-done

### Highlights

- No linter errors ✅ 
- Correct GitFlow followed ✅ 
- Used (JavaScript, HTML, CSS) best practices ✅ 
- Good descriptive PR✅ 
- Commits are meaningful ✅ 
- Working App ✅ 
- Correct .gitignore file ✅ 
- Correct implementation of "Components, Lifting state up, Hooks" ✅ 

### **Optional Suggestions**
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

**Pull request merged**